src/CMakeLists.txt

```diff
@@ -58,6 +58,14 @@ if(UMF_BUILD_OS_MEMORY_PROVIDER)
     endif()
 endif()
 
+if(UMF_BUILD_ZE_MEMORY_PROVIDER)
+    set(UMF_SOURCES_LINUX ${UMF_SOURCES_LINUX}
+        provider/provider_ze_memory.c)
+    if(LINUX)
```

Comment:
Why is this linux-specific? L0 also works on windows

Reply:
Yes, but at the moment we don't have the internal infrastructure to test it